.github/workflows/run-deploy.yaml

```diff
@@ -133,7 +137,7 @@ jobs:
         with:
           username: ${{steps.get-lightsail-username.outputs.USERNAME}}
           server: ${{steps.get-lightsail-ip-address.outputs.IP_ADDRESS}}
-          ssh_private_key: ${{ secrets.SSH_PRIVATE_KEY }}
+          ssh_private_key: ~/.ssh/lightsail.pem
```

Comment:
I think the `ssh_private_key` parameter expects the key itself instead of a path to a key